blueocean-web/src/main/js/try.js

```diff
@@ -1,6 +1,19 @@
 var $ = require('jquery-detached').getJQuery();
 var jsModules = require('@jenkins-cd/js-modules');
 
+function getAppUrl() {
+    var rootUrl = jsModules.getRootURL();
+    
+    // If there's no prefix, rooturl is blank, resulting in js-modules
+    // returning the static url. We don't want to use that when
+    // switching to blue ocean, so return '' instead.
+    if (rootUrl.indexOf('/static/') === 0) {
```

Comment:
@kzantow We don't actually want the static url ... js-modules just falls back to giving that when the root url attribute is not set for some reason e.g. when there's no context path, or on older versions of Jenkins that just don't set it at all. This is ok for what js-modules does, but I'll just remove it and access the DOM directly. Gimme a min 😄